src/types/JobParameters.ts

```diff
@@ -6,13 +6,12 @@ export interface IJobParameters<DATA = unknown | void> {
 
 	name: string;
 	priority: number;
-	nextRunAt: Date | null;
+	nextRunAt?: Date | null;
```

Comment:
now `| null` can be reverted again right?

Reply:
We never $unset nextRunAt and there are multiple places where nextRunAt is set explicitly to null
The reason to make nextRunAt optional was the delete of nextRunAt in line 297 in JobDbRepository.ts

Reply:
ah right `nextRunAt: Date | null;`

there is .toJson() used (which is kinda weird anyway), but the reason behind this is that it returned a partial result before, you have changed the result of it to a non partial return value, what was the motivation behind that? Could we just let it return a partial result? Otherwise we coudl just change line 252 to:
from `...job.toJson()` to `...(job.toJson() as Partial<IJobParameters>),`

Reply:
I reverted that ? making nextRunAt mandatory in typings. I changed in line 297 of JobDbRepository.ts the typing when deleting that field. This makes imho the most sense.